bundle/manifests/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -802,6 +1054,46 @@ spec:
     targetPort: 9443
     type: ValidatingAdmissionWebhook
     webhookPath: /validate-redhatcop-redhat-io-v1alpha1-randomsecret
+  - admissionReviewVersions:
```

Comment:
not sure what is going on here, but these will be added during the bundle build and should not be here

Reply:
Removed.